requirements-dev.txt

```
@@ -1,12 +1,17 @@
 # Internationalization and Localization requirements
--e git://github.com/edx/xblock-sdk.git@v0.1.6#egg=xblock-sdk==v0.1.6
-Django==1.11.25
+-e 'git://github.com/edx/xblock-sdk.git@v0.1.6#egg=xblock-sdk==v0.1.6 ; python_version == "2.7"'
+-e 'git://github.com/edx/xblock-sdk.git@0.2.2#egg=xblock-sdk==v0.2.2 ; python_version > "2.7"'
+Django==1.11.25; python_version == '2.7'
```

Instead of pinning to an exact release we can do this to allow bugfixes.
```suggestion
Django<1.12; python_version == '2.7'
```